packages/cfd/src/Stores/Modules/CFD/Helpers/useCfdStores.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { createContext, PropsWithChildren, useContext } from 'react';
+import { useStore } from '@deriv/stores';
+import type { TCFDStore } from '../../../../types/cfd-store.types';
```

Comment:
Better create an alias for our store types in our webpack config.

Reply:
Answered above